Driver/Driver/Extenison/JailbreakChecker.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  JailbreakChecker.swift
+//  Driver
+//
+//  Created by Turan Çabuk on 26.02.2023.
+//
+
+import Foundation
+
+import UIKit
+
+import UIKit
+
+/// This class contains functions that check whether the device that wants to login to the application has a jailbreak or not.
+class JailbreakChecker {
+    
+    /**
+     This function checks whether the device that wants to login contains jailbreak libraries and returns a Bool value.
+     ## Important Notes ##
+     This func returns a Bool value to allow or block the device in the appDelegate.
+     */
+    func isJailbroken() -> Bool {
+        let fileManager = FileManager.default
```

Comment:
App Delegate veya SceneDelegate çağıracağımız için burada **fileManager** oluşturmamız problem değil. Ama başka bir yerde kullanıyor olsaydık her bu fonksiyon çağrıldığında **fileManager** oluşmuş olacaktı, buda boşuna sistem gereksinimi demek.